app/src/components/ProfileEdit/QuestionPage.tsx

```diff
@@ -130,24 +142,41 @@ export const QuestionPage = (props: Props) => {
 
     return (
         <MuiThemeProvider theme={theme}>
-            <div>
-                {props.stepwise && (
-                    <>
-                        <LinearProgress
-                            variant="determinate"
-                            value={(state.groupIndex / groups.length) * 100}
-                        />
-                        <Box my={3}></Box>
+            <div style={{ position: 'relative' }}>
+                <div style={{ marginBottom: 80 }}>
+                    {props.stepwise && (
                         <LinearProgress
                             variant="determinate"
                             value={
-                                (state.subgroupIndex /
-                                    groups[state.groupIndex].length) *
-                                100
+                                state.submitPage
+                                    ? 100
+                                    : getStepperProgress() * 100
                             }
                         />
-                    </>
-                )}
+                    )}
+                    {stepper.map((question, i) => {
```

Comment:
along with the change mentioned above, this statement can be simplified in a later iteration.